api/services/lau.spec.ts

```diff
@@ -176,7 +175,6 @@ describe('createAccessLog', () => {
       action: 'CREATED',
       timestamp: '2023-09-06T12:34:56.789Z',
       reason: 'mock-reason',
-      requestStartTimestamp: '2023-09-06T10:00:00.000Z',
```

Comment:
I don't think you should remove this field. How will you represent when specific access is granted for a future period - e.g. for a week, in a month's time

Reply:
Unless it is some future feature coming up, there isn't a way to ask or give permissions for specific timeframe in the future. The only option is to approve from now until the chosen time. And on LAU side it is always going to be "now" or even more confusingly "midnight of today (in the past)", even if permission was granted only during the day